associate-software-engineer/fc-1/javascript/modulo-2/topico-2/ex-19.js

```diff
@@ -0,0 +1,13 @@
+const palavras = ['gato', 'cachorro', 'gato', 'pássaro'];
+
+const words = new Map();
+
+const contando = palavras.forEach(word => {
```

Comment:
forEach é `in place` ele retorna `void` ou seja não tem retorno. Não existe pq armazenar em uma variável